tests/server/auth/fixtures/jwt_auth.ini

```diff
@@ -0,0 +1,6 @@
+[mlflow]
+default_permission = READ
+database_uri = sqlite:///basic_auth.db
+admin_username = admin
+admin_password = password
+authorization_function = tests.server.auth.jwt_auth:authenticate_request
```

Comment:
I am a little bit concerned about putting this in the config file. For example, if a user want to plug in their own jwt implementation and reuse the existing mlflow authorization, they would need to modify `basic_auth.ini` and then still run `mlflow ui --app-name=basic-auth`.

Reply:
Can you explain the concern in more detail? Are you saying the above wouldn't work, or that it's confusing, or something else? Can you suggest potential changes that you'd like to see?

At this point, I don't have enough information to understand or address your concern.

Reply:
@gabrielfu: Please let me know what you'd like to do here, also with your concern about the test. I'm happy to make changes but not sure what you'd like me to do. A few ideas I thought of:
- Create an abstract authorization app with two concrete implementations:
  - basic-auth
  - configurable-auth
  - ??
 
To me, it adds some technical complexity that I'm not sure is justified, but I'm flexible. Part of my thinking is, the auth feature is still new, and many basic features are yet to be added (e.g. groups) Perhaps it could be better to focus on some new user-facing features now and then rework the engineer/Ops-facing bits later once it is more feature-complete.